repos/system_upgrade/el7toel8/actors/updateetcsysconfigkernel/tests/unit_test.py

```diff
@@ -0,0 +1,24 @@
+import os
+import tempfile
+
+from leapp.libraries.actor import library
+
+def test_update_kernel_config(monkeypatch):
```

Comment:
this test is failing

Reply:
Hey, thanks a lot for reviewing. This error is unexpected, it seems to be a problem with leapp.db missing during tests. I am not able to reproduce it locally, lets see what happens on another run of it.

Reply:
So we added a pytest.mark.skip decorator to this test so we can proceed with the review of this MR. The error is unexpected and we are still debugging to understand better where it is. I will create another PR to remove the skip and we can track the problem there

Reply:
It’s because of the audit functions they need database access and I guess the directory /var/lib/leapp does not exist on the test systems

Reply:
Hey @vinzenz, thanks for taking a look at this. Any idea why only this specific test is triggering the problem?